### PR TITLE
Per-feed CSS class for default newsfeed module

### PR DESCRIPTION
**Pull request by ninjabreadman**
- Added CSS class support for default newsfeed module as requested on the forum here (and elsewhere): https://forum.magicmirror.builders/topic/6014/change-colors-on-each-news-feed
- Updated default module config with second feed (I feel this may overcomplicate default config)
- Updated `CHANGELOG.md` and the module's `README.md`, some sample CSS, and an e2e test for the class
- Added all default module CSS to `Gruntfile.js`, fixed the consequent linting errors

First PR to MM, so certainly welcome all feedback.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -31,8 +31,7 @@ module.exports = function(grunt) {
 					"css/main.css",
 					"modules/default/calendar/calendar.css",
 					"modules/default/clock/clock_styles.css",
-					"modules/default/currentweather/currentweather.css",
-					"modules/default/weatherforcast/weatherforcast.css"
+					"modules/default/**/*.css"
```
Comment (roramirez) [inline Gruntfile.js]: This change is not part the request to introduce in this PR, you should be move in another PR.

File: modules/default/alert/ns-default.css [hunk 2]
```diff
@@ -74,6 +74,7 @@
   0% {
     -webkit-transform: perspective(1000px) rotate3d(1, 0, 0, -90deg);
   }
+
```
Comment (roramirez) [inline modules/default/alert/ns-default.css]: This is not part of the PR.

File: modules/default/newsfeed/newsfeed.js [hunk 3]
```diff
@@ -8,13 +8,18 @@
...
 
 Module.register("newsfeed",{
-
 	// Default module config.
 	defaults: {
 		feeds: [
 			{
 				title: "New York Times",
 				url: "http://www.nytimes.com/services/xml/rss/nyt/HomePage.xml",
+				className: "nyt",
```
Comment (roramirez) [inline modules/default/newsfeed/newsfeed.js]: You can set this in the example configuration file, not here because this affect another instalations.

File: tests/configs/modules/newsfeed/default.js [hunk 4]
```diff
@@ -26,7 +26,8 @@ var config = {
 				feeds: [
 					{
 						title: "Rodrigo Ramirez Blog",
-						url: "http://localhost:8080/tests/configs/data/feed_test_rodrigoramirez.xml"
+						url: "http://localhost:8080/tests/configs/data/feed_test_rodrigoramirez.xml",
```
Comment (roramirez) [inline tests/configs/modules/newsfeed/default.js]: Include this in another tests case, not change the present.

File: tests/e2e/modules/newsfeed_spec.js [hunk 5]
```diff
@@ -36,5 +36,9 @@ describe("Newsfeed module", function() {
 		it("show title newsfeed", function() {
 			return app.client.waitUntilTextExists(".newsfeed .small", "Rodrigo Ramirez Blog", 10000).should.be.fulfilled;
 		});
+
```
Comment (roramirez) [inline tests/e2e/modules/newsfeed_spec.js]: Include in separte test case.

File: modules/default/newsfeed/newsfeed.js [hunk 6]
```diff
@@ -44,6 +49,10 @@ Module.register("newsfeed",{
 		return ["moment.js"];
 	},
 
+	getStyles: function() {
```
Comment (roramirez) [inline modules/default/newsfeed/newsfeed.js]: I personally should not set this behaviour here. The custom css for users instaations should be in the custom CSS file.

**Comment by MichMich** [comment]
Thanks for your contribution. @roramirez has highlighted some important issues with the PR. If you're able to fix these, I'll definitely merge this PR.

Cheers!

File: Gruntfile.js (hunk 1 above)
Comment (ninjabreadman) [reply Gruntfile.js]: This was an incidental change when adding `newsfeed` CSS to `stylelint` ... wasn't clear why some module CSS shouldn't lint, except that it's more of a dependency than source file. But if I'm reverting CSS addition, so will remove this as well.

File: modules/default/alert/ns-default.css (hunk 2 above)
Comment (ninjabreadman) [reply modules/default/alert/ns-default.css]: As above; only lint fixes (nothing substantive) given change to `Gruntfile.js`. Will revert.

**Comment by MichMich** [comment]
@roramirez Good to go?

**Comment by ninjabreadman** [comment]
Not quite ... I'll need to either fix or remove the test for the newsfeed class. Thanks.

**Comment by roramirez** [comment]
@MichMich not yet...

**Comment by MichMich** [comment]
Shout my name when you guys are good to go! ;)

**Comment by MichMich** [comment]
@ninjabreadman Is this something you want to fix? Or do you want me to close this PR?

**Comment by ninjabreadman** [comment]
Please close it – I need to take another look, but haven’t had time. If I can get it passing, I’ll resubmit once it’s ready.

**Comment by MichMich** [comment]
Thanks!
